Resources/views/frontend/_public/src/js/components.js

```diff
@@ -66,16 +66,25 @@ if (cardToken !== undefined) {
     };
 
     const disableForm = () => {
-        submitButton.disabled = true;
+        let submitButtons = document.querySelectorAll('button[type="submit"]');
+        if (submitButtons.length) {
+            submitButtons.forEach(function(el) {
+                el.disabled = true;
+            });
+        }
     };
 
     const enableForm = () => {
-        submitButton.disabled = false;
+        let submitButtons = document.querySelectorAll('button[type="submit"]');
+        if (submitButtons.length) {
```

Comment:
please modify your condition like following to avoid possible implicit typecast issues

if (submitButtons.length > 0)